### ISSUE TITLE
Octave 7.1: subsref doctest fails

### ISSUE THREAD

**Issue by cbm755** [cbm755/octsympy#1112]
```
@sym/subsref ......................................... FAIL    1/4   

   >> A.flat

      expected:    Matrix([[10, 11, 12]])

      got     : ??? subsref: function called with too many outputs
```
This might indicate some deeper bug.

**Comment by cbm755** [comment]
See here, seems we should just varagout this function.

https://sourceforge.net/p/octave/interval/ci/d3d6819e990eba8bf6363bc9f2d3d2cc78dcff03/

https://savannah.gnu.org/bugs/?func=detailitem&item_id=61898

**Comment by cbm755** [comment]
Fixed by #1113.
